systemd/eyesy-python.service

```diff
@@ -4,7 +4,7 @@ Description=Python Pygame Video Engine
 [Service] 
 Type=simple
 WorkingDirectory=/home/we/sidekick/patches/Eyesy/engines/python
-ExecStart=nohup python main.py
+ExecStart=nohup python -u main.py
```

Comment:
This change is to tell python to write to STDOUT unbuffered; without this, messages are chunked, which makes for a bad developer experience. We want STDOUT messages to appear in the log immediately.